src/engraving/playback/playbackmodel.cpp

```diff
@@ -892,6 +892,13 @@ PlaybackModel::TickBoundaries PlaybackModel::tickBoundaries(const ScoreChangesRa
             result.tickFrom = std::min(result.tickFrom, firstTiedNote->tick().ticks());
             result.tickTo = std::max(result.tickTo, lastTiedNote->tick().ticks());
         }
+        if (item->parent() && item->parent()->isChord()) {
```

Comment:
Originally I had an "else" in here, but that didn't work in the case of "undo", where the ornament object being modified doesn't have a valid parent at this point.  Technically this code could probably be moved inside the "`if (item->isNote()) {` block above, but it's safer enough here, given the only purpose is to ensure that the end-tick of the playback update region covers the longest possible trill spanner that might be relevant.